mods/ra/maps/allies-10b/allies10b.lua

```diff
@@ -0,0 +1,323 @@
+				local victim2 = Reinforcements.Reinforce(USSR, { "e2" }, { GrenEntry.Location, DiePoint2.Location })
+				Trigger.OnIdle(victim2[1], function(two)
+					two.Kill("BulletDeath")
+					Media.PlaySound("gun5.aud")
+				end)
+			end)
+			Trigger.AfterDelay(DateTime.Seconds(6), function()
+				local tanya = Reinforcements.Reinforce(Greece, { "e7.noautotarget" }, { TanyaEntry.Location, TankRoomCam.Location })
+				TanyaArrived = true
+				TanyaSurvive = AddPrimaryObjective(Greece, "tanya-survive")
+				Media.PlaySoundNotification(Greece, "lefty")
+				Trigger.OnKilled(tanya[1], function()
+					Greece.MarkFailedObjective(TanyaSurvive)
+				end)
+			end)
+			Trigger.AfterDelay(DateTime.Minutes(1), function()
+				tankCam.Destroy()
+			end)
+		end
+	end)
+end
+
+TanyaObjectiveCheck = function()
+	if TanyaArrived then
+		Greece.MarkCompletedObjective(TanyaSurvive)
+	end
+end
+
+DeactivateMissiles = function()
+	Trigger.OnAllKilled(Engineers, function()
+		Greece.MarkFailedObjective(StopNukes)
+	end)
+
+	Trigger.OnEnteredFootprint(TopLeftConsole, function(actor, id)
+		if actor.Type == "e6" and not TopLeftTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			TopLeftTriggered = true
+			Media.PlaySpeechNotification(Greece, "ControlCenterDeactivated")
+			Media.DisplayMessage(UserInterface.Translate("nuclear-missile-deactivated"), UserInterface.Translate("console"))
+		end
+	end)
+
+	Trigger.OnEnteredFootprint(BottomLeftConsole, function(actor, id)
+		if actor.Type == "e6" and not BottomLeftTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			BottomLeftTriggered = true
+			Media.PlaySpeechNotification(Greece, "ControlCenterDeactivated")
+			Media.DisplayMessage(UserInterface.Translate("nuclear-missile-deactivated"), UserInterface.Translate("console"))
+		end
+	end)
+
+	Trigger.OnEnteredFootprint(TopRightConsole, function(actor, id)
+		if actor.Type == "e6" and not TopRightTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			TopRightTriggered = true
+			Media.PlaySpeechNotification(Greece, "ControlCenterDeactivated")
+			Media.DisplayMessage(UserInterface.Translate("nuclear-missile-deactivated"), UserInterface.Translate("console"))
+		end
+	end)
+
+	Trigger.OnEnteredFootprint(MiddleRightConsole, function(actor, id)
+		if actor.Type == "e6" and not MiddleRightTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			MiddleRightTriggered = true
+			Media.PlaySpeechNotification(Greece, "ControlCenterDeactivated")
+			Media.DisplayMessage(UserInterface.Translate("nuclear-missile-deactivated"), UserInterface.Translate("console"))
+		end
+	end)
+end
+
+ticked = TimerTicks
+Tick = function()
+	if Greece.HasNoRequiredUnits() then
+		Greece.MarkFailedObjective(StopNukes)
+	end
+
+	if TopLeftTriggered and BottomLeftTriggered and TopRightTriggered and MiddleRightTriggered then
+		Greece.MarkCompletedObjective(StopNukes)
+		Greece.MarkCompletedObjective(Paperclip)
+		TanyaObjectiveCheck()
+	end
+
+	if ticked > 0 then
```

Comment:
```suggestion
	if Ticked > 0 and (Ticked % DateTime.Seconds(1)) == 0 then
```

so we only update the translation when necessary.

Reply:
Clock doesn't tick down when we do this.

Reply:
Figured it out (I think)